client/my-sites/plugins/plugin-automated-transfer/wpadmin-auto-login.jsx

```diff
@@ -0,0 +1,11 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+function getPixelUrl( siteUrl ) {
+	const pixel = encodeURI( siteUrl + 'wp-includes/images/blank.gif' );
+	return `${ siteUrl }/wp-login.php?redirect_to=${ pixel }`;
```

Comment:
Do we actually login on a GET action? 

Let me know if I'm missing steps, but after transferring a site and hitting wp-admin, I still see a prompt to connect.

![screen shot 2017-02-17 at 10 05 01 am](https://cloud.githubusercontent.com/assets/1270189/23077341/069ac054-f4f9-11e6-8617-a3c089d1095c.png)

![screen shot 2017-02-17 at 10 05 09 am](https://cloud.githubusercontent.com/assets/1270189/23077356/10c034ec-f4f9-11e6-81b0-ba06da1c3423.png)